src/frontend/components/SøknadsSteg/Dokumentasjon/Dokumentasjon.tsx

```diff
@@ -118,15 +116,13 @@ const Dokumentasjon: React.FC = () => {
                     />
                 ))}
             {innsendingStatus.status === RessursStatus.FEILET && <Feilside />}
-            {toggles.DISABLE_SEND_INN_KNAPP && (
-                <div>
-                    <AlertStripe variant="error">
-                        <BodyShort>
-                            Denne siden er under utvikling og du kan derfor ikke sende inn en søknad
-                        </BodyShort>
-                    </AlertStripe>
-                </div>
+
+            {/*
```

Comment:
Jeg tenker at vi kanskje ikke trenger eksempel på hvordan man bruker den her? 🤔 Hvis vi vil ha med det eksempelet kunne vi kanskje puttet det i ReadMe i stedet for?